f5-openstack-agent-dist/scripts/construct_setups.py

```diff
@@ -287,7 +288,7 @@ def get_args():
 interprets their meaning before returning such content.
     """
     expected = ['working_directory', 'operating_system', 'version']
-    possible = ['json'].extend(expected)
+    possible = ['json'] + expected
```

Comment:
Was there a particular reason for this change?

Reply:
['json'].extend(expected) has no return value, variable "possible" is always None.

Reply:
If we do not use construct_setup.py as a command line for the test, it should not be an issue.